CHANGELOG.md

```diff
@@ -3,7 +3,91 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 This file follows the formats and conventions from [keepachangelog.com]
 
-## [Unreleased]
+## [2.8.0] 2019-06-18
```

Comment:
This needs to be a link to a diff between two tags - see the bottom of the CHANGELOG.md file.